client/src/components/Comment.js

```diff
@@ -71,8 +85,10 @@ class Comment extends Component {
   render() {
     return (
       <div className="comment">
-        <span style={{ fontWeight: 'bold' }}>{this.props.userName}</span>
-        <span style={{ fontSize: '12px' }}>&nbsp;&bull;&nbsp;{this.findTime()}</span>
+        <span className="comment-name" style={{ fontWeight: 'bold' }}>
```

Comment:
comment-name class referenced but `.comment-name {}` SCSS class occurs nowhere in the codebase.

Pull `fontWeight` out the style attribute of the span into a `comment-name {}` SCSS class.
Its best to keep all static styling out of the jsx

Reply:
👍